src/index.js

```diff
@@ -85,15 +86,18 @@ export class SlideDeck extends React.Component {
     Provider: DefaultProvider,
     width: '100vw',
     height: '100vh',
-    ignoreKeyEvents: false
+    ignoreKeyEvents: false,
+    autoTransition: -1
   }
 
   state = {
     length: this.props.slides.length,
     index: 0,
     mode: modes.normal,
     notes: {},
-    step: -1
+    fragments: {},
```

Comment:
NB: it seems like it might be good to combine notes/fragments/etc into a single object for the current slide, but can definitely be a followup thing